test/helpers/export-csv-helper.js

```diff
@@ -2,8 +2,8 @@ module.exports.TEAM_CASELOAD_RESULT = {
   title: 'Test Team',
   caseloadDetails: {
     overallCaseloadDetails: [
-      { linkId: 2767, name: 'John Smith', gradeCode: 'PO', untiered: 0, d2: 3, d1: 6, c2: 9, c1: 6, b2: 12, b1: 18, a: 9, totalCases: 189, caseType: 'COMMUNITY' },
-      { linkId: 2771, name: 'Tony Test', gradeCode: 'PO', untiered: 0, d2: 3, d1: 6, c2: 9, c1: 6, b2: 12, b1: 18, a: 9, totalCases: 189, caseType: 'COMMUNITY' }
+      { linkId: 2767, name: 'John Smith', gradeCode: 'PO', a: 9, b1: 18, b2: 12, c1: 6, c2: 9, d1: 6, d2: 3, untiered: 0, totalCases: 189, caseType: 'COMMUNITY' },
```

Comment:
Use case type constant for test data?

Reply:
Don't think that this is necessary here.  This data is just to ensure that what is output is what we expect, so we could actually put anything in here.

Reply:
👍